breacharbiter.go

```diff
@@ -1245,15 +1223,6 @@ type RetributionStore interface {
 	// is aware of any breaches for the provided channel point.
 	IsBreached(chanPoint *wire.OutPoint) (bool, error)
 
-	// Finalize persists the finalized justice transaction for a particular
-	// channel.
-	Finalize(chanPoint *wire.OutPoint, finalTx *wire.MsgTx) error
```

Comment:
nice to see some of this db code removed! another artifact left over from the pre-height-hint era